mpi_learn/train/data.py

```diff
@@ -55,26 +55,25 @@ def set_file_names(self, file_names):
         else:
             self.file_names = file_names
             
-    def generate_data(self):
+    def generate_data(self, load_tensor=False):
```

Comment:
it might be better to make H5TData(H5Data) which re-implement load_data and concat_data

Reply:
@thongonary can you actually do that ?